src/main/java/dinkplugin/util/ItemUtils.java

```diff
@@ -121,16 +122,35 @@ public SerializedItemStack stackFromItem(ItemManager itemManager, int id, int qu
         return new SerializedItemStack(id, quantity, price, composition.getName());
     }
 
-    public String formatStack(SerializedItemStack item) {
-        return String.format("%d x %s (%s)", item.getQuantity(), item.getName(), QuantityFormatter.quantityToStackSize(item.getTotalPrice()));
+    public String formatStack(SerializedItemStack item, PriceType priceType) {
```

Comment:
Given that `formatStack` just reads the price contained within `SerializedItemStack`, it seems that `PriceType` param should be applied to `stackFromItem` and `formatStack` could just be passed a boolean to dictate whether the price should be included

That said, a simpler approach is to remove `PriceType` (just rely on the RL wiki price setting) & have a setting for the include-price boolean

Reply:
The reason for PriceType over boolean is if we'd want to add high alch pricing instead of GE pricing

Reply:
It could still be a boolean, with an additional setting deciding item price source